scene/resources/particles_material.cpp

```diff
@@ -1170,6 +1339,20 @@ void ParticlesMaterial::_validate_property(PropertyInfo &property) const {
 	if (property.name.begins_with("orbit_") && !particle_flags[PARTICLE_FLAG_DISABLE_Z]) {
 		property.usage = PROPERTY_USAGE_NONE;
 	}
+
+	if (turbulence_mode == 0) {
+		if (property.name == "turbulence_power" ||
+				property.name == "turbulence_influence_over_life" ||
+				property.name == "turbulence_scale" ||
+				property.name == "turbulence_speed" ||
+				property.name == "turbulence_position_influence" ||
+				property.name == "turbulence_velocity_influence" ||
+				property.name == "turbulence_scale_influence" ||
+				property.name == "turbulence_rotation_influence" ||
+				property.name == "turbulence_color_ramp") {
+			property.usage = PROPERTY_USAGE_NONE;
+		}
```

Comment:
I think this should be removed, it prevents resetting these properties to the default values.